src/lib.ts

```diff
@@ -61,13 +61,15 @@ export default async function isMergable(opts: Options): Promise<Result> {
 	let pullRequestNumber = pullRequest.data.number
 	let pullRequestTitle = pullRequest.data.title
 	let pullRequestState = pullRequest.data.state
-	let pullRequestCreator = pullRequest.data.user.login
+	let pullRequestCreator = pullRequest.data.user
+		? pullRequest.data.user.login
+		: "unknownCreator"
 	let pullRequestCommits = pullRequest.data.commits
 	let pullRequestBaseRef = pullRequest.data.base.ref
 	let pullRequestHeadRef = pullRequest.data.head.ref
 	let pullRequestUrl = pullRequest.data.html_url
-	let pullRequeseGitMergeable = pullRequest.data.mergeable
-	let pullRequeseGitRebaseable = pullRequest.data.rebaseable
+	let pullRequeseGitMergeable = pullRequest.data.mergeable || false
+	let pullRequeseGitRebaseable = pullRequest.data.rebaseable || false
```

Comment:
These are now nullable, so I put in some defaults that are probably not amazing.

Reply:
`??` ?